static/app/gettingStartedDocs/react-native/react-native.tsx

```diff
@@ -92,253 +123,284 @@ Sentry.mobileReplayIntegration({
-            <Fragment>
-              <p>
-                {tct(
-                  '[wizardLink:Sentry Wizard] will patch your project accordingly, though you can [setupManuallyLink:setup manually] if you prefer.',
-                  {
-                    wizardLink: (
-                      <ExternalLink href="https://github.com/getsentry/sentry-wizard" />
-                    ),
-                    setupManuallyLink: (
-                      <ExternalLink href="https://docs.sentry.io/platforms/react-native/manual-setup/manual-setup/" />
-                    ),
-                  }
-                )}
-              </p>
-              <List symbol="bullet">
-                <ListItem>
-                  {t(
-                    'iOS Specifics: When you use Xcode, you can hook directly into the build process to upload debug symbols and source maps.'
-                  )}
-                </ListItem>
-                <ListItem>
-                  {tct(
-                    "Android Specifics: We hook into Gradle for the source map build process. When you run [code:./gradlew] assembleRelease, source maps are automatically built and uploaded to Sentry. If you have enabled Gradle's [code:org.gradle.configureondemand] feature, you'll need a clean build, or you'll need to disable this feature to upload the source map on every build by setting [code:org.gradle.configureondemand=false] or remove it.",
-                    {
-                      code: <code />,
-                    }
-                  )}
-                </ListItem>
-              </List>
-            </Fragment>
-          ),
-        },
-      ],
-    },
-  ],
-  configure: params => [
-    {
-      type: StepType.CONFIGURE,
-      configurations: [
-        ...(params.isProfilingSelected
-          ? [
+const onboarding: OnboardingConfig<PlatformOptions> = {
+  install: params =>
+    isAutoInstall(params)
+      ? [
+          {
+            type: StepType.INSTALL,
+            description: tct(
+              'Run [code:@sentry/wizard] to automatically configure your project:',
+              {code: <code />}
+            ),
+            configurations: [
+              {
+                language: 'bash',
+                code: `npx @sentry/wizard@latest -s -i reactNative --org ${params.organization.slug} --project ${params.projectSlug}`,
+              },
               {
-                description: t(
-                  'React Native Profiling is available since SDK version 5.32.0.'
+                description: (
+                  <Fragment>
+                    <p>
+                      {t(
+                        'The Sentry wizard will automatically patch your project with the following:'
+                      )}
+                    </p>
+                    <List symbol="bullet">
+                      <ListItem>
+                        {t(
+                          'Configure the SDK with your DSN and performance monitoring options'
```

Comment:
The wizard doesn't do any performance changes at the moment.

````suggestion
                          'Configure the SDK with your DSN'
````